odoo/sql_db.py

```diff
@@ -545,7 +618,13 @@ def autocommit(self, on):
     def commit(self):
         """ Perform an SQL `COMMIT` """
         self.flush()
-        self._cursor.execute('SAVEPOINT "%s"' % self._savepoint)
+        # FIXME: if this is enabled, /website:WithContext.test_search fails
+        #        trying to release/rollback a savepoint which doesn't exist
+        #        anymore, has to run the test-set of runbot (fails both on
+        #        runbot and locally), /website alone works fine
```

Comment:
Thanks for leaving that useful info!
Can you make the commit message consistent with that?